README.md

```diff
@@ -117,13 +117,14 @@ Set `$GOPATH` variable to point to your Go workspace directory and add `$GOPATH/
 6. Run the trillian-map server 
 
   ```sh
-docker-compose up trillian-map -d
+docker-compose up -d trillian-map
   ```
 
 7. Provision a log and a map 
 ```sh
-go run $GOPATH/src/github.com/google/trillian/cmd/createtree/main.go --admin_server=localhost:8090 --pem_key_path=testdata/log-rpc-server.privkey.pem --pem_key_password="towel" --signature_algorithm=ECDSA --tree_type=LOG
-go run $GOPATH/src/github.com/google/trillian/cmd/createtree/main.go --admin_server=localhost:8090 --pem_key_path=testdata/log-rpc-server.privkey.pem --pem_key_password="towel" --signature_algorithm=ECDSA --tree_type=MAP
+MAP_IP=`docker inspect keytransparency_trillian-map_1 | grep -E '"IPAddress": ' | tail -n 1 | cut -d \" -f 4`
```

Comment:
Looks like the format command is a bit more canonical way to do this:
`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' container_name_or_id`

https://stackoverflow.com/questions/17157721/how-to-get-a-docker-containers-ip-address-from-the-host